nh_clinical/activity_extension.py

```diff
@@ -658,9 +781,7 @@ def get_recurring_activity_date_scheduled(
             due.
         :rtype: str
         """
-        activity_model = self.env['nh.activity']
-        frequency = activity_model.sudo()\
-            .browse(triggered_activity_id).data_ref.frequency
+        frequency = activity.data_ref.frequency
         date_scheduled = (dt.now() + td(minutes=frequency)).strftime(DTF)
         return date_scheduled
```

Comment:
Should we make this `api.mutli` style instead? Then it's just `self.frequency` with no need to access data_ref.

Reply:
This is a static method due to the fact the calling method of the calling method is passed an activity ( `trigger_policy_change_state`).

This method was there to take the frequency from the activity's linked model and get the scheduled date using it